fog-azure-rm.gemspec

```
@@ -22,8 +22,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest', '~> 5.8.4'
   spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'codeclimate-test-reporter' , '~> 1.0.0'
-  spec.add_dependency 'fog-core', '~> 1.43'
-  spec.add_dependency 'fog-json', '~> 1.0.2'
+  spec.add_dependency 'fog-core', '~> 2.1.0'
```

Let's try and make it backwards compatible - I suggest removing version restriction.

```suggestion
  spec.add_dependency 'fog-core'
```